Sources/_StringProcessing/Documentation.docc/README.md

```diff
@@ -0,0 +1,41 @@
+# ``_StringProcessing/Regex``
```

Comment:
What does this file do?

Reply:
This is for organizing how the different symbols defined in the `Regex` type are organized on the `Regex` documentation page. By default, symbols are grouped in sections by kind: "Initializers", "Instance Properties", etc.

It looks like I didn't change this file name — I'll update that.